stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1
+apiVersion: {{ template "rabbitmq.workloadApiVersion" . }}
```

Comment:
Charts only supports k8s N and N-1 (when N = 1.9 as the latest version supported by most providers). We'd prefer not to have extensive logic determining the API version to use, so we should just set this to `app/v1beta2`.

Reply:
According to the README this chart should support k8s 1.4+ :wink:

Reply:
Hmm good catch, that may need to be updated. The reason the maintainers have decided on supporting N and N-1 only is to avoid issues like this where charts end up having lots of template hacks to support older k8s versions (e.g. switching from annotations to fields in specs are a usual culprit). I can't find any guidelines where we state the N & N-1 claim, so I think we need to reevaluate where the maintainers stand on this and put the decision in the guidelines.

cc @kubernetes/charts-maintainers

Reply:
As long as I can get it running with 1.8 ^^

Reply:
Aha - found it (https://github.com/kubernetes/charts#supported-kubernetes-versions). @goruha could you set the apiVersion to just `apps/v1beta2` here? Thanks!